### DISCUSSION TITLE
Author filtering is broken

### DISCUSSION THREAD

**Discussion by GermainBergeron** [community/community#38016] [Projects and Issues] [answered]
When I try to filter on the author in the PRs of any repository I see all PRs and not just the one from that author. I have the same issue on the https://github.com/pulls page where I see that I created more than 56,000,000 PRs which is obviously wrong.

I've confirmed with a few members of my team and they all see something similar

<img width="1094" alt="image" src="https://user-images.githubusercontent.com/260007/199321711-c90bb25c-51df-4ddb-8343-2c8576f74376.png">

**Comment by airtower-luna** [comment]
I'm seeing the same thing, pretty clearly a bug! :sweat_smile:

**Comment by john-light** [comment]
Can confirm this same behavior when using both `author:john-light` (my username) and `author:@me` filters:

<img width="516" alt="Screenshot 2022-11-01 160141" src="https://user-images.githubusercontent.com/9424721/199329514-3de236f5-c820-4b60-858a-a8bdcb146110.png">

<img width="602" alt="Screenshot 2022-11-01 160116" src="https://user-images.githubusercontent.com/9424721/199329652-10dbe955-3796-48fe-94ff-545cfcf674e3.png">

**Comment by GermainBergeron** [comment, marked as answer]
There is an incident on the status page now https://www.githubstatus.com/incidents/gw33xm0hbl00

**Comment by david-engelmann** [comment]
I'm able to get it to filter correctly by clicking into the search bar and running the command that is pulled up when you enter the pr or issue section
